op-guide/pd-control.md

```diff
@@ -0,0 +1,206 @@
+# PD Control instructions
+
+As a command line tool of PD, PD Control is used for obtaining the cluster state information and tuning the cluster.
+
+## Source code compiling
```

Comment:
we don't want to user build pd-ctl, `pd-ctl` exists in the release binary already.